### DISCUSSION TITLE
I use supabase connect timescaleDB And is that right?

### DISCUSSION THREAD

**Discussion by Antony-Jia** [supabase/supabase#7960] [General]
I use docker-compose to connect my postgrel(timescaledb) like this:

db:
    container_name: supabase-db
    image: timescale/timescaledb-ha:pg14-latest
    restart: unless-stopped
    ports:
      - ${POSTGRES_PORT}:5432
    environment:
      POSTGRES_USER: ${POSTGRES_USER}
      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
    volumes:
      - ./volumes/db/data:/var/lib/postgresql/data
      - ./volumes/db/init:/docker-entrypoint-initdb.d

when I modify some config and make it run. I met the log print very often

the log like this:
supabase-db        | 2022-07-30 13:53:47.030 UTC [40] LOG:  logical decoding found consistent point at 0/1B64388
supabase-db        | 2022-07-30 13:53:47.030 UTC [40] DETAIL:  There are no running transactions.
supabase-db        | 2022-07-30 13:53:47.030 UTC [40] STATEMENT:  with pub as (
supabase-db        |            select
supabase-db        |              concat_ws(
supabase-db        |                ',',
supabase-db        |                case when bool_or(pubinsert) then 'insert' else null end,
supabase-db        |                case when bool_or(pubupdate) then 'update' else null end,
supabase-db        |                case when bool_or(pubdelete) then 'delete' else null end
supabase-db        |              ) as w2j_actions,
supabase-db        |              coalesce(
supabase-db        |                string_agg(
supabase-db        |                  realtime.quote_wal2json(format('%I.%I', schemaname, tablename)::regclass),
supabase-db        |                  ','
supabase-db        |                ) filter (where ppt.tablename is not null),
supabase-db        |                ''
supabase-db        |              ) w2j_add_tables
supabase-db        |            from
supabase-db        |              pg_publication pp
supabase-db        |              left join pg_publication_tables ppt
supabase-db        |                on pp.pubname = ppt.pubname
supabase-db        |            where
supabase-db        |              pp.pubname = $1
supabase-db        |            group by
supabase-db        |              pp.pubname
supabase-db        |            limit 1
supabase-db        |          ),
supabase-db        |          w2j as (
supabase-db        |            select
supabase-db        |              x.*, pub.w2j_add_tables
supabase-db        |            from
supabase-db        |              pub,
supabase-db        |              pg_logical_slot_get_changes(
supabase-db        |                $2, null, $3,
supabase-db        |                'include-pk', '1',
supabase-db        |                'include-transaction', 'false',
supabase-db        |                'include-timestamp', 'true',
supabase-db        |                'write-in-chunks', 'true',
supabase-db        |                'format-version', '2',
supabase-db        |                'actions', pub.w2j_actions,
supabase-db        |                'add-tables', pub.w2j_add_tables
supabase-db        |              ) x
supabase-db        |          )
supabase-db        |          select
supabase-db        |            xyz.wal,
supabase-db        |            xyz.is_rls_enabled,
supabase-db        |            xyz.subscription_ids,
supabase-db        |            xyz.errors
supabase-db        |          from
supabase-db        |            w2j,
supabase-db        |            realtime.apply_rls(
supabase-db        |              wal := w2j.data::jsonb,
supabase-db        |              max_record_bytes := $4
supabase-db        |            ) xyz(wal, is_rls_enabled, subscription_ids, errors)
supabase-db        |          where
supabase-db        |            w2j.w2j_add_tables <> ''
supabase-db        |            and xyz.subscription_ids[1] is not null

and very often

**Comment by silentworks** [comment]
I'm not sure what the question is here? can you edit the original message and state what you are asking please?

**Reply by Antony-Jia** [reply]
> I'm not sure what the question is here? can you edit the original message and state what you are asking please?

I modify the question.

The main questions is postgres log below is very large and often:

supabase-db | 2022-07-30 13:53:47.030 UTC [40] LOG: logical decoding found consistent point at 0/1B64388
supabase-db | 2022-07-30 13:53:47.030 UTC [40] DETAIL: There are no running transactions.

**Reply by w3b6x9** [reply]
@Antony-Jia those logs are related to Realtime and can be ignored. Is it possible for you to set log statement level to `none`?

**Reply by Antony-Jia** [reply]
Thank you for reply , and I do it now. I think it's better to make a switch to control.
